templates/data/generic_module_data.yaml

```diff
@@ -57,6 +56,61 @@ templates:
         jobConfigs:
           - repoName: github.com/kyma-project/keda-manager
             jobs:
+              - jobConfig:
+                  name: "keda-operator-nightly-periodic"
+                  cron: "0 6 * * 1-5" # "Runs on 7 am every day-of-week from Monday through Friday."
+                  annotations:
+                    description: Keda nightly verify
+                    pipeline.platform: gke
+                    pipeline.installer: kyma deploy
+                    pipeline.clusterprovisioning: gcloud cli
+                    pipeline.type: integration
+                    testgrid-dashboards: keda-nightly
+                    testgrid-days-of-results: "60"
+                  path_alias: github.com/kyma-project/kyma
+                  cluster: "trusted-workload"
+                  labels:
+                    preset-kyma-keyring: "true"
+                    preset-kyma-encryption-key: "true"
+                    preset-build-main: "true"
+                    preset-nightly-github-integration: "true"
+                    preset-kms-gc-project-env: "true"
+                    preset-certificates-bucket: "true"
+                    preset-slack-alerts: "true"
```

Comment:
Do you need this many presets in this job? Please inject only presets that you actually need.